### PR TITLE
Feature/styling

### PR DESCRIPTION
**Pull request by mershark**
In this repository, you'll find the following updates:
1. I've incorporated any missing JSX elements into my project, ensuring that I haven't introduced any additional functionality beyond the specified requirements.
2. The final design of my project closely adheres to Zeplin's design guidelines, encompassing all required elements and styling to ensure a consistent and visually appealing presentation.
3.  I've diligently applied the CSS rules outlined in Zeplin to style the website, maintaining fidelity to the provided design specifications.

### CODE REVIEW

File: src/components/navigation.jsx [hunk 1]
```diff
@@ -1,18 +1,25 @@
...
+  <header>
+    <nav>
+      <div className="nav-container">
+        <h2 className="logo">Bookstore CMS</h2>
+        <div>
+          <Link className="nav books" to="/">Books</Link>
+        </div>
+        <div>
+          <Link className="nav category" to="/category">Category</Link>
+        </div>
```
Comment (victorokoroji) [inline src/components/navigation.jsx]: - [x] Good job so far 👍🏽 . I think you should kindly increase the font size of your navigations and capitalize the font to match the  Zeplin design

Your design 👇🏽 
image

Zeplin design 👇🏽 
image

File: src/components/book.jsx [hunk 2]
```diff
@@ -1,19 +1,44 @@
...
+          <button type="button" className="button comment">
+            comments
+          </button>
+          <button type="button" className="button" onClick={onClick}>
             remove
           </button>
+          <button type="button" className="button">
+            edit
+          </button>
+        </div>
```
Comment (victorokoroji) [inline src/components/book.jsx]: - [x] KIndly increase the spacing between the actions buttons to match the zeplin design 💯 

Zeplin design 👇🏽 
image

your design 👇🏽 
image

File: src/components/book.jsx [hunk 3]
```diff
@@ -1,19 +1,44 @@
...
+              completed
+            </p>
+          </div>
+        </div>
+        <div className="chapter-div">
+          <p className="chapter current">CURRENT CHAPTER</p>
+          <p className="chapter number">Chapter 17</p>
+          <button type="button" className="update">
+            update progress
+          </button>
```
Comment (victorokoroji) [inline src/components/book.jsx]: - [x] I think you should increase the padding around this button to match the zeplin dessign 💯 

your design 👇🏽 
image

zeplin design 👇🏽 
image

File: src/components/bookForm.jsx [hunk 4]
```diff
@@ -33,17 +33,17 @@ function BookForm() {
   return (
     <form>
       <h2>ADD NEW BOOK</h2>
```
Comment (victorokoroji) [inline src/components/bookForm.jsx]: - [x] This title is not strictly black in color but gray, kindly use the right color

zeplin design 👇🏽 
image

your design 👇🏽 
image

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @mershark, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the Git Flow ✅ 
- Good UI design✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ademibowale — APPROVED** [review]
Hi @mershark  , 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️
- Use functional components✔️
- Descriptive commit messages ✔️ 
- Good use of Github flow ✔️






- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**